src/common/components/layouts/pages/results/summary/Summary.js

```diff
@@ -21,10 +22,11 @@ const JOB_END_STATUS = {
     TIMEOUT: 'timeout',
 };
 
-function Summary({ fileInfo, resultSummary, jobEndStatus }) {
+function Summary({ fileInfo, selectedProfile, resultSummary, jobEndStatus }) {
     return (
         <Paper className="summary">
-            <h2>{fileInfo.name}</h2>
+            <FileName fileInfo={fileInfo} component={'h2'} size={'max'} />
+            <p>{selectedProfile}</p>
```

Comment:
Use the map received from backend (validationProfiles) to get `humanReadableName` for the `selectedProfile`. Display the `humanReadableName` property here instead of the `selectedProfile` code.
E.g. for the `WCAG_2_1_COMPLETE` this label should say `WCAG 2.1 (All)`